### PR TITLE
Rename long fields to askCount / minCount / ansCount

### PR DESCRIPTION
**Pull request by sorawit**
CC @tansawit @evilpeach. Scan will need to update to reflect this by next week.

### CODE REVIEW

File: chain/x/oracle/client/cli/tx.go [hunk 1]
```diff
@@ -21,17 +21,17 @@ import (
...
-	flagClientID                 = "client-id"
-	flagSchema                   = "schema"
-	flagSourceCodeURL            = "url"
+	flagName          = "name"
+	flagDescription   = "description"
+	flagScript        = "script"
+	flagCallFee       = "call-fee"
+	flagOwner         = "owner"
+	flagCalldata      = "calldata"
+	flagAskCount      = "requested-validator-count"
```
Comment (taobun) [inline chain/x/oracle/client/cli/tx.go]: Need to change?

File: chain/x/oracle/client/cli/tx.go [hunk 2]
```diff
@@ -122,10 +122,10 @@ $ %s tx oracle request 1 --calldata 1234abcdef --requested-validator-count 4 --s
 	}
 
 	cmd.Flags().BytesHexP(flagCalldata, "c", nil, "Calldata used in calling the oracle script")
-	cmd.Flags().Int64P(flagRequestedValidatorCount, "r", 0, "Number of top validators that need to report data for this request")
-	cmd.MarkFlagRequired(flagRequestedValidatorCount)
-	cmd.Flags().Int64P(flagSufficientValidatorCount, "v", 0, "Minimum number of reports sufficient to conclude the request's result")
-	cmd.MarkFlagRequired(flagSufficientValidatorCount)
+	cmd.Flags().Int64P(flagAskCount, "r", 0, "Number of top validators that need to report data for this request")
```
Comment (taobun) [inline chain/x/oracle/client/cli/tx.go]: Need to change short name of this field (r doesn't make sense anymore)

**Review by taobun — CHANGES_REQUESTED** [review]

File: chain/x/oracle/client/cli/tx.go (hunk 1 above)
Comment (sorawit) [reply chain/x/oracle/client/cli/tx.go]: Removed :P

File: chain/x/oracle/client/cli/tx.go (hunk 2 above)
Comment (sorawit) [reply chain/x/oracle/client/cli/tx.go]: Removed :P

**Review by taobun — APPROVED** [review]

**Pull request merged**